### PR TITLE
Fix scripts to deploy sk4 on staging

### PR DESCRIPTION
**Pull request by petuhovskiy**
Create `init_safekeeper.sh` inspired by `init_pageserver.sh` and adjust ansible inventory for the new instance.

Safekeeper will be initialized after this PR will be merged to the main.

ref https://github.com/zenithdb/console/pull/970

### CODE REVIEW

File: .circleci/ansible/scripts/init_safekeeper.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+# get instance id from meta-data service
+INSTANCE_ID=$(curl -s http://169.254.169.254/latest/meta-data/instance-id)
```
Comment (ololobus) [inline .circleci/ansible/scripts/init_safekeeper.sh:4]: I wonder why we don't use some domain name here as well

**Review by ololobus — APPROVED** [review]
LGTM, but maybe @cicdteam has some comments

File: .circleci/ansible/scripts/init_safekeeper.sh (hunk 1 above)
Comment (petuhovskiy) [reply .circleci/ansible/scripts/init_safekeeper.sh:4]: I guess we use instance_id as something that is fixed and unique, this way it's better than domain names.

Comment (ololobus) [reply .circleci/ansible/scripts/init_safekeeper.sh:4]: Yeah, but I mean this IP 169.254.169.254

Comment (petuhovskiy) [reply .circleci/ansible/scripts/init_safekeeper.sh:4]: It's a special AWS IP for obtaining metadata, I don't see that it has domain name https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html

**Pull request merged**